sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/Generated/Models/ServerlessComputeSettings.cs

```diff
@@ -0,0 +1,70 @@
+        /// <para>
+        /// To assign an object to the value of this property use <see cref="BinaryData.FromObjectAsJson{T}(T, System.Text.Json.JsonSerializerOptions?)"/>.
+        /// </para>
+        /// <para>
+        /// To assign an already formatted json string to this property use <see cref="BinaryData.FromString(string)"/>.
+        /// </para>
+        /// <para>
+        /// Examples:
+        /// <list type="bullet">
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson("foo")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("\"foo\"")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        private IDictionary<string, BinaryData> _serializedAdditionalRawData;
+
+        /// <summary> Initializes a new instance of <see cref="ServerlessComputeSettings"/>. </summary>
+        public ServerlessComputeSettings()
+        {
+        }
+
+        /// <summary> Initializes a new instance of <see cref="ServerlessComputeSettings"/>. </summary>
+        /// <param name="serverlessComputeCustomSubnet"> The resource ID of an existing virtual network subnet in which serverless compute nodes should be deployed. </param>
+        /// <param name="serverlessComputeNoPublicIP"> The flag to signal if serverless compute nodes deployed in custom vNet would have no public IP addresses for a workspace with private endpoint. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal ServerlessComputeSettings(ResourceIdentifier serverlessComputeCustomSubnet, bool? serverlessComputeNoPublicIP, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            ServerlessComputeCustomSubnet = serverlessComputeCustomSubnet;
+            ServerlessComputeNoPublicIP = serverlessComputeNoPublicIP;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> The resource ID of an existing virtual network subnet in which serverless compute nodes should be deployed. </summary>
+        public ResourceIdentifier ServerlessComputeCustomSubnet { get; set; }
+        /// <summary> The flag to signal if serverless compute nodes deployed in custom vNet would have no public IP addresses for a workspace with private endpoint. </summary>
+        public bool? ServerlessComputeNoPublicIP { get; set; }
```

Comment:
Better to rename to `ServlessComputeHasNoPublicIP` or just `HasNoPublicIP`

Reply:
fixed